src/assets/styles/_chart-container.scss

```diff
@@ -1,27 +1,49 @@
- 
-
-$block-class: 'd3-container';
+$block-class: 'chart-container';
 
 @at-root {
     .#{$block-class} {
         & &__overview-member-chart {
             margin: 0 -1rem;
         }
 
-        main-chart {
+        &__main-chart {
             width: 100;
+            height: 80vh;
+            margin-top:4rem;
+        }
+
+        & &__chart {
+            width: 95%;
+            height: 100%;
+            margin: auto;
         }
 
         & &__chart-bar {
             float: right;
-            margin-top: 0.625rem;
+            margin-right: 4.7rem;
+            margin-top: -.10rem;
+            &--timeline{
+                z-index: 12;
+                position: relative;
+            }
+            &--filter{
+                z-index: 12;
+                position: relative;
+            }
+        }
+        & &__custom-icon {
+            color:red,
```

Comment:
The dark theme being implemented later on will not be able to affect this hard coded 'red'. Please integrate the color into MUI.